app/javascript/controllers/auto_citation_controller.js

```diff
@@ -0,0 +1,96 @@
+import { Controller } from "stimulus";
+
+export default class extends Controller {
+  static targets = ["titleField", "contributorFirst", "contributorLast", "contributorRole",
+    "year", "month", "day", "manual", "auto", "switch"];
+
+  connect() {
+    this.purl = this.data.get("purl") || ":link:" // Use a real purl on a persisted item or a placeholder
```

Comment:
Do/should we include any hints in the UI so the user knows to leave `:link:` alone for later? Or not necessary?

Reply:
That's specified in the tool tip in the h2 design documentation, but we haven't gotten to tooltips yet.

Reply:
@jcoyne OK, works for me.